http_check/check.py

```diff
@@ -169,7 +169,11 @@ def _load_conf(self, instance):
         http_response_status_code = str(instance.get('http_response_status_code', DEFAULT_EXPECTED_CODE))
         timeout = int(instance.get('timeout', 10))
         config_headers = instance.get('headers', {})
-        headers = agent_headers(self.agentConfig)
+        default_headers = instance.get("include_default_headers", True)
```

Comment:
Can we use enforce that the result is a boolean with `_is_affirmative`, c.f. https://github.com/DataDog/integrations-core/blob/master/disk/check.py#L60-L61 ?